edgexfoundry/app-service-configurable
# refactor!: Rename command line flags for the sake of consistency

BREAKING CHANGE: renamed -c/--confdir to -cd/--configDirand -f/--file to -cf/--configFile

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  https://github.com/edgexfoundry/edgex-docs/pull/916

## Testing Instructions
Build service
run `./app-service-configurable  -h`
verify flags in help have new names
run `./app-service-configurable --confdir something`
verify error due to --confdir unknown
run `./app-service-configurable --configDir something`
verify error due to file `something` not found

## New Dependency Instructions (If applicable)
